Branch-SDK/Branch-SDK/NSMutableDictionary+Branch.h

```diff
@@ -10,7 +10,7 @@
 #import <Foundation/Foundation.h>
 
 
-void ForceNSMutableDictionaryToLoad(void);
+void ForceNSMutableDictionaryToLoad(void) __attribute__((constructor));
 
 
 @interface NSMutableDictionary (Branch)
```

Comment:
I think it might be possible to remove this whole category and just use subscript assignment, e.g. `dict[@"key"] = nil;` is just fine, I believe. This would also include the `safeSetValue` methods here and there in the SDK. I don't think it does any harm, but less is more.